src/python/phenix_apps/apps/scorch/mm/README.md

```diff
@@ -20,6 +20,7 @@ metadata:
     capture:
       bridge: phenix
       convert: true # convert to JSON (for parsing into Elastic, for example) (default is false)
+  - type: reset_cc  # delete all miniccc commands and responses
```

Comment:
@GhostofGoes we have a `cc` scorch component... I think it would be cleaner to add a `reset` type to it instead of here.

Reply:
@activeshadow The `cc` component as it's currently written only operates on individual VMs. I could update the component to add stage-level commands like the `mm` component currently has? Thoughts?

Reply:
Changes are pushed!